core/internal/cltest/cltest.go

```diff
@@ -165,6 +165,10 @@ func MustRandomBytes(t *testing.T, l int) (b []byte) {
 	return b
 }
 
+func FormatWithPrefixedChainID(chainID, id string) string {
```

Comment:
You could pass through `any` args (that are expected to be string or implement String()), to drop the `String()` calls.
```suggestion
func FormatWithPrefixedChainID(chainID, id any) string {
```